git-pr

```diff
@@ -131,7 +131,7 @@ name as the remote branch name (only works if you have a local
 branch).
 
 If "-f" is *first* argument, then force push.  If "-r" is after that,
-automatically make a PR (using "git pr gh").  `-d` and `-n` as in the `gh`
+automatically make a PR (using "git pr gh"). "-d" and "-n" as in the "gh"
```

Comment:
Unrelated to this PR, but FYI option parsing was improved and now they can be in any order and possibly combined, but still have to be before arguments.